sdk/src/main/java/com/qonversion/android/sdk/QUserPropertiesManager.kt

```diff
@@ -37,7 +38,9 @@ class QUserPropertiesManager @Inject internal constructor(
     }
 
     fun onAppForeground() {
-        sendPropertiesWithDelay(retryDelay)
+        if (propertiesStorage.getProperties().isNotEmpty()) {
+            sendPropertiesWithDelay(retryDelay)
+        }
```

Comment:
I think it would be better if we move this check into the `sendPropertiesWithDelay` method as we can call it anywhere else without this check and face this bug again